File: easy/distribution_of_sweets/distribution_of_sweets.py

def distribution(siblings,popsicles):
	'''returns a string "give away" if multiple sweets can be distributed among a group of children,
	otherwise returns the string "eat them yourself"'''
	
	if popsicles%siblings == 0:
		print("give away")
	else:
		print("eat them yourself")
		
siblings = input("Number of brothers")
popsicles = input("number of sweets")

distribution(siblings,popsicles)

File: easy/distribution_of_sweets/distribution_of_sweets.kt
fun distribution(siblings:Int,brothers:Int):String{
    
    if(siblings%siblings == 0){
            return("give away")
    }

    return("eat them yourself")
}	


fun main(args: Array<String>){
	
    println("Number of brothers : ")
    var brothers = readLine()!!.toInt()

    println("Number of sweets : ")
    var siblings = readLine()!!.toInt()

   	println(distribution(siblings,brothers))
  
}
